src/Microsoft.AspNetCore.Server.Kestrel.Core/Internal/Http/Frame.FeatureCollection.cs

```diff
@@ -75,187 +76,91 @@ private void ExtraFeatureSet(Type key, object value)
-            get
-            {
-                return QueryString;
-            }
-
-            set
-            {
-                QueryString = value;
-            }
+            get => QueryString;
+            set => QueryString = value;
         }
 
         string IHttpRequestFeature.RawTarget
         {
-            get
-            {
-                return RawTarget;
-            }
-            set
-            {
-                RawTarget = value;
-            }
+            get => RawTarget;
+            set => RawTarget = value;
         }
 
         IHeaderDictionary IHttpRequestFeature.Headers
         {
-            get
-            {
-                return RequestHeaders;
-            }
-
-            set
-            {
-                RequestHeaders = value;
-            }
+            get => RequestHeaders;
+            set => RequestHeaders = value;
         }
 
         Stream IHttpRequestFeature.Body
         {
-            get
-            {
-                return RequestBody;
-            }
-
-            set
-            {
-                RequestBody = value;
-            }
+            get => RequestBody;
+            set => RequestBody = value;
         }
 
         int IHttpResponseFeature.StatusCode
         {
-            get
-            {
-                return StatusCode;
-            }
-
-            set
-            {
-                StatusCode = value;
-            }
+            get => StatusCode;
+            set => StatusCode = value;
         }
 
         string IHttpResponseFeature.ReasonPhrase
         {
-            get
-            {
-                return ReasonPhrase;
-            }
-
-            set
-            {
-                ReasonPhrase = value;
-            }
+            get => ReasonPhrase;
+            set => ReasonPhrase = value;
         }
 
         IHeaderDictionary IHttpResponseFeature.Headers
         {
-            get
-            {
-                return ResponseHeaders;
-            }
-
-            set
-            {
-                ResponseHeaders = value;
-            }
+            get => ResponseHeaders;
+            set => ResponseHeaders = value;
         }
 
         Stream IHttpResponseFeature.Body
         {
-            get
-            {
-                return ResponseBody;
-            }
-
-            set
-            {
-                ResponseBody = value;
-            }
+            get => ResponseBody;
+            set => ResponseBody = value;
         }
 
         CancellationToken IHttpRequestLifetimeFeature.RequestAborted
         {
-            get
-            {
-                return RequestAborted;
-            }
-            set
-            {
-                RequestAborted = value;
-            }
+            get => RequestAborted;
+            set => RequestAborted = value;
         }
 
         bool IHttpResponseFeature.HasStarted
         {
-            get { return HasResponseStarted; }
+            get => HasResponseStarted;
```

Comment:
These can be reduced to `bool IHttpResponseFeature.HasStarted => HasResponseStarted;` right?